File: backend/controllers/dtos/tracked_locationDTO.go
package dtos

type TrackedLocationDTO struct {
	Id           int64
	Lat          float64
	Lon          float64
	StartTime    int64
	EndTime      int64
	Name         string
	ReasonId     int64
	Confirmed    bool
	TrackedDayId int64
	Movements    []TrackedMovementDTO
}

File: frontend/src/app/Models/DTOs/TrackedLocationDTO.ts
import { TrackedMovementDTO } from "./TrackedMovementDTO";

export class TrackedLocationDTO {
    Lat: number;
    Lon: number;
    StartTime: number;
    EndTime: number;
    Name: string;
    Reason: string;
    TrackedMovements: TrackedMovementDTO[];
}